replay-tests/helloworld/test/ui/hello/HelloWorldSpec.java

```diff
@@ -11,5 +13,8 @@ public class HelloWorldSpec extends BaseSpec {
   public void openHelloWorldPage() {
     open("/");
     $("h1").shouldHave(text("Hello, world!"));
+    $("#repeat").shouldHave(text("Repeat 666 times")).click();
+    $("h1").shouldHave(text("Hello, world #666!"));
+    $("#link-to-image").shouldHave(attribute("href", "/app/public/img/banksy-kyiv-7.jpeg")).click();
```

Comment:
Are you too lazy? Or just testing if I'm a good reviewer? :grinning: 

```suggestion
    $("#link-to-static-file").shouldHave(href("/public/img/banksy-kyiv-7.jpeg")).click();
```

Reply:
fixed